packages/ra-core/src/controller/create/useCreateController.ts

```diff
@@ -182,7 +187,7 @@ export const useCreateController = <
     return {
         isFetching: false,
         isLoading: false,
-        isPending: saving,
+        isPending: isPendingCanAccess,
```

Comment:
Should be something like `isPendingAuthentication || isPendingAccessControl`

Reply:
As discussed, it's actually not needed